src/lib.rs

```diff
@@ -412,6 +418,15 @@ to advance the applied index inside.
 
 For more information, check out an [example](examples/single_mem_node/main.rs#L113-L179).
 
+Sometimes it's better not to block the raft machine in IO operation, so that latency of
+read/write can be more predictable and the fsync frequencey can be controlled. The crate
+supports async ready to offload the IO operation to other thread. The usage is the same as
+above except:
+1. all writes are not required to be persisted, they can be written into memory caches;
```

Comment:
```suggestion
1. Before handling the next write, all previous writes are not required to be persisted, they can be written into memory caches;
```

Reply:
What's the difference?

Reply:
`all writes are not required to be persisted` looks like no need to persist the write at all.

Reply:
How about "all writes are not required to be persisted immediately"?

Reply:
lgtm